tests/test_free_energy.py

```diff
@@ -423,13 +413,18 @@ def test_get_water_sampler_params(num_windows):
     # Use the simple charges simply because it is faster
     forcefield = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
     st = SingleTopology(mol_a, mol_b, core, forcefield)
-    solvent_sys, solvent_conf, solvent_box, solvent_top = builders.build_water_system(
-        3.0, forcefield.water_ff, mols=[mol_a, mol_b]
+    host_config = builders.build_water_system(3.0, forcefield.water_ff, mols=[mol_a, mol_b])
+
+    # (YTZ): dedup later with rbfe.Host
```

Comment:
@mcwitt  - I need to figure out how to make HostConfig Immutable later so I can dedup  with rbfe.Host but I don't want this PR to get any larger.